### DISCUSSION TITLE
Running gateway-cli with out running it on fedimint CLI tool

### DISCUSSION THREAD

**Discussion by randoman14** [fedimint/fedimint#6150] [Q&A] [answered]
Hello im wondering if Can I run the (gateway-cli) command outside of the emulator CLI tool (just mprocs)? 

Or running the fedimint-cli?

Thanks!

**Comment by dpc** [comment]
Both tools are meant for standalone use, and `just mprocs` setup is just for dev work.

**Reply by randoman14** [reply]
So (just mprocs) is not the problem its something with my configuration?

(nix:nix-shell-env) frank@frank:~/fedimint$ gateway-cli
bash: gateway-cli: command not found

**Reply by dpc** [reply, marked as answer]
Well, you need to have `gateway-cli` available on your system. Depending on the setup, this might be installed in different ways. Releases on github even come with deb and rpm packages, and even statically linked binaries. Just make sure you're using the right version for your gateway.

See Discord and #mint-ops channel if you'd prefer more interactive help.

**Reply by randoman14** [reply]
Alight cool thanks!
